source/features/extend-diff-expander.tsx

```diff
@@ -5,7 +5,7 @@ import features from '../libs/features';
 function expandDiff(event) {
 	// Skip if the user clicked directly on the icon
 	if (!event.target.closest('.js-expand')) {
-		select('.js-expand', event.delegateTarget).click();
+		select<HTMLElement>('.js-expand', event.delegateTarget).click();
```

Comment:
Used the generic `HTMLElement` as I have no idea what kind of element this is supposed to be. I would guess a `<div />` maybe an `<a href="..." />`, but not certain.

Reply:
`HTMLAnchorElement`